server/events/models/models.go

```diff
@@ -476,12 +476,15 @@ const (
 	// PlannedPlanStatus means that a plan has been successfully generated but
 	// not yet applied.
 	PlannedPlanStatus
-	// ErrorApplyStatus means that a plan has been generated but there was an
+	// ErroredApplyStatus means that a plan has been generated but there was an
 	// error while applying it.
 	ErroredApplyStatus
 	// AppliedPlanStatus means that a plan has been generated and applied
 	// successfully.
 	AppliedPlanStatus
+	// NotPlannedPlanStatus means that either no plan has been generated yet or
+	// a plan has been deleted - eg after using the UI to unlock the project
+	NotPlannedPlanStatus
```

Comment:
This is only set when the project is unlocked, not when no plan has been generated yet, so maybe we can call this `DiscardedPlanStatus` and update the description to say:
```
means that there was an unapplied plan that was discarded due to a project being unlocked
```